src/scripts/OSFramework/Helper/Device.ts

```diff
@@ -335,10 +342,10 @@ namespace OSFramework.Helper {
 		public static get IsIphoneWithNotch(): boolean {
 			if (DeviceInfo._isIphoneWithNotch === undefined) {
 				// get the device pixel ratio
-				const ratio = window.devicePixelRatio || 1;
+				const ratio = (sessionStorage.pixelRatio ? sessionStorage.pixelRatio : window.devicePixelRatio) || 1;
```

Comment:
The same here: `sessionStorage.previewDevicesPixelRatio`

Reply:
Done